src/components/ShoeImagePreview.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div class="bg-grey-lighten-4" :style="{ position: 'relative' }">
+    <v-img
+      aspect-ratio="16/9"
+      cover
+      src="/src/assets/shoe.png"
```

Comment:
@balanejacob Define the images in variables in the parent component and pass them to this component via props.